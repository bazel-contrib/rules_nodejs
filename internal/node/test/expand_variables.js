const fs = require('fs');
const args = process.argv.slice(2);
const outfile = args.shift();
args.push(process.env['SOME_OTHER_ENV']);
fs.writeFileSync(outfile, JSON.stringify(args, null, 2), 'utf-8');
