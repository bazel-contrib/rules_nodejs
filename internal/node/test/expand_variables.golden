[
  "some_value",
  "some_other_value"
]